src/components/SelectCTypes/SelectCTypes.tsx

```diff
@@ -127,7 +127,7 @@ class SelectCTypes extends React.Component<Props, State> {
     return {
       baseValue: `${cType.cType.hash}`,
       label: <CTypePresentation cTypeHash={cType.cType.hash} />,
-      value: `${cType.cType.metadata.title.default} ${cType.cType.hash}`,
+      value: `${cType.cType.schema.$id} ${cType.cType.hash}`,
```

Comment:
Same as above

Reply:
since `ctype` is of type `CTypeMetadata`, we should be able to get the title from the metadata

Reply:
@LeonFLK plz check this